bin/delete_lambda.py

```diff
@@ -1,18 +1,21 @@
 import subprocess
 import json
 
-# Gets user input for the prefix of lambdas to delete
+# Gets user input for the prefix of ORCA lambdas to delete
 prefix = input("Enter Prefix: ")
 
+# Gets AWS Profile to use for ORCA lambda deletion
+profile = input("Enter AWS Profile: ")
```

Comment:
I am confused why we need a profile here. I think this can be removed.

Reply:
Removed